Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+authors = ["MaidSafe Developers <dev@maidsafe.net>"]
+description = "Implementation of a DKG"
+documentation = ""
+edition = "2018"
+exclude = [""]
+homepage = "https://maidsafe.net"
+license = "GPL-3.0"
+name = "bls_dkg"
+readme = "README.md"
+repository = "https://github.com/maidsafe/BLS-DKG"
+version = "0.1.0"
+
+[dependencies]
+bincode = "1.1.4"
+failure = "~0.1.5"
+lazy_static = "~1.2.0"
```

Comment:
Be great if we can use const fn rather than lazy_static as it's a nightmare dependency at times.

Reply:
Cheers @madadam  I had not realised the semver implications here. Nice one

Reply:
Here it's explained in more details: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html. Basically we should be using the "caret requirements" instead of "tilde requirements" for crates that are past v1.0 to avoid unnecessary version conflicts.

Reply:
I got this removed already.
Shall I change it back?